### ISSUE TITLE
Apathy cursor out of sync with display cursor

### ISSUE THREAD

**Issue by mattcodez** [coopermaruyama/apathy-theme#21]
I have a strange issue that just popped up in the last couple days. When going to edit a file, the display cursor would not be positioned where the code edits are happening. Seems to happen with any file. It can even get more complex when in the context of braces and parens and starts to get really broken. I effectively can't work with Atom in this state. 

I tested that 'atom --safe' works however. I then went through and disabled all my custom plug-ins and the problem remained. I then narrowed down that changing the syntax theme resolved the issue. In the gif below, I'm first editing with the Atom Dark syntax theme and all is well. I then switch to Apathy and the cursor is broken.

![apathy-atom-io](https://cloud.githubusercontent.com/assets/2740549/10972118/c3aa7160-83a5-11e5-8df7-28c8eebe7542.gif)

Not sure why this would all of a sudden happen just in the last 48 hours. Especially as this package appears to not have had any updates in months. Disabling custom plugins doesn't help either.

**Comment by czxtm** [comment]
Hi, this is most likely due to the feature that lets you add padding on the left side. Unfortunately this feature has been causing more issues than it's worth, so I released a new update today that removes it until I can come up with a better solution (and probably in its own package). 

Try updating and let me know if the issue is still happening.

**Comment by czxtm** [comment]
I just pushed one more commit in v1.8.1 which has some fixes for #20. Atom would measure characters in fonts too early when using external fonts and they would often cause issues like this. It looks like atom recently fixed this, and also removed some methods we were using in order to force re-measurements. It might have something to do with this issue too, since like you said this package wasn't updated for a while.

In any case, I'm quite confident the new version should work for you. If not, please reopen this. Also, if it happens again, do me a favor and run in dev mode (View > Developer > Open in Dev Mode), then open up developer tools (cmd + alt + i) and copy/paste the output of running the following code in the console:

```
apathy = atom.packages.getActivePackage('apathy-theme').mainModule
apathy.apathyView.getDebugLog()
```

This runs a bunch of debugging utilities I've added in this theme's code which will make it a lot easier for me to debug.

Thanks a lot for using my theme - Please reopen this if v1.8.1 doesn't fix it!

**Comment by mattcodez** [comment]
Seems to work great now, thanks!
